fluxc/src/test/java/org/wordpress/android/fluxc/network/UserAgentTest.kt

```diff
@@ -0,0 +1,45 @@
+package org.wordpress.android.fluxc.network
+
+import android.webkit.WebSettings
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.mockStatic
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+import org.wordpress.android.util.PackageUtils
+import kotlin.test.assertEquals
+
+private const val APP_NAME = "App Name"
+private const val USER_AGENT = "Default User Agent"
+private const val APP_VERSION = "1.0"
+
+@RunWith(RobolectricTestRunner::class)
+class UserAgentTest {
+    private val context = RuntimeEnvironment.application.applicationContext
+
+    @Test
+    fun testUserAgent() = withMockedPackageUtils {
+        mockStatic(WebSettings::class.java).use { settings ->
+            settings.`when`<Any> { WebSettings.getDefaultUserAgent(context) }.thenReturn(USER_AGENT)
```

Comment:
I'd suggest using `whenever` instead of `when` because 'when' is a reserved keyword in Kotlin.

```suggestion
            whenever(WebSettings.getDefaultUserAgent(context)).thenReturn(USER_AGENT)
```

Reply:
Thank you for the suggestion @jarvislin 🙇 
I hadn't realised that I could use whenever in the `mockStatic` context. I changed this with b760c39e369040ee0cf400555b21edab8ba0ea9b